storage/bucket.go

```diff
@@ -1892,6 +1925,52 @@ func customPlacementFromProto(c *storagepb.Bucket_CustomPlacementConfig) *Custom
 	return &CustomPlacementConfig{DataLocations: c.GetDataLocations()}
 }
 
+func (a *Autoclass) toRawBucketAutoclass() *raw.BucketAutoclass {
```

Comment:
Can leave "bucket" out of these method names (so `toRawAutoclass` and `toProtoAutoclass`)